test/recipes/81-test_cmp_cli.t

```diff
@@ -200,7 +200,7 @@ sub test_cmp_cli_aspect {
             }
         }
     };
-    unlink "test.cert.pem", "test.cacerts.pem", "test.extracerts.pem";
+    # not unlinking "test.cert.pem", "test.cacerts.pem", "test.extracerts.pem"
```

Comment:
I would suggest ensuring that each case outputs different files as well.  It's fine if you want to do that in a separate PR

Reply:
> I would suggest ensuring that each case outputs different files as well. It's fine if you want to do that in a separate PR

Done in a further commit of this PR.

Reply:
Uhmmm, did you push that commit?

Reply:
Yes, the commit differentiating the output files is commit bd185484de5fa1564d32786eb480654e5fe990b3.

Reply:
Its main changes are in `test/recipes/81-test_cmp_cli_data/test_enrollment.csv`
while the changes to other files are minor test case improvements I came across on this occasion.